src/services/app-versions-service.ts

```diff
@@ -25,6 +25,7 @@ export const listAppVersionsByAppId = async (appId: AppId): Promise<Array<AppVer
     return sortedAppVersions;
   } catch (error: any) {
     if (error instanceof HttpError) {
+      logger.error(error.message);
```

Comment:
I would limit the print to 4XX errors and not all errors

Reply:
You need to bump the CLI version as part of this PR

Reply:
Bumped version.